addons/mail/controllers/webclient.py

```diff
@@ -11,13 +11,13 @@ class WebclientController(http.Controller):
     @http.route("/mail/init_messaging", methods=["POST"], type="json", auth="public")
     @add_guest_to_context
     def mail_init_messaging(self):
-        if not request.env.user.sudo()._is_public():
-            return request.env.user.sudo(request.env.user.has_group("base.group_portal"))._init_messaging()
+        if not request.env.user._is_public():
+            return request.env.user.sudo(False)._init_messaging()
```

Comment:
Note/reminder: env.user has a sudo context as True by default (I think that's pretty dumb in terms of security)